README.md

```diff
@@ -466,6 +466,35 @@ To build an Amazon-like autocomplete menu, suggesting popular queries and for th
 }
 ```
 
+##### User-generated data: protecting against XSS
```

Comment:
I would remove the "user-generated data", and use this title: "XSS Protection" so that everyone reads it no matter what.